lua/packer/compile.lua

```diff
@@ -439,14 +439,26 @@ local function make_loaders(_, plugins, output_lua, should_profile)
         end
       end
 
-      if plugin.module then
+      if plugin.module or plugin.module_pattern then
         loaders[name].only_sequence = false
         loaders[name].only_setup = false
-        if type(plugin.module) == 'string' then
-          plugin.module = { plugin.module }
-        end
-        for _, module_name in ipairs(plugin.module) do
-          module_lazy_loads['^' .. vim.pesc(module_name)] = name
+
+        if plugin.module then
+          if type(plugin.module) == 'string' then
+            plugin.module = { plugin.module }
+          end
+
+          for _, module_name in ipairs(plugin.module) do
+            module_lazy_loads['^' .. vim.pesc(module_name)] = name
+          end
+        else
```

Comment:
Another very minor point but this doesn't seem to explicitly also verify that `plugin.module_pattern` exists before operating on it, it only seems to check if module exists, and otherwise it always runs this branch? Seems like the type conditional won't break, and I'm assuming the `ipairs` can handle a nil value, but still seems a little dangerous. Should this not be `elseif plugin.module_pattern`

Reply:
Yep, good point.

Reply:
Makes sense, missed the parent conditional 👍🏿